java/vector/src/main/codegen/templates/BaseWriter.java

```diff
@@ -90,6 +90,7 @@
     void clear();
     void copyReader(FieldReader reader);
     MapWriter rootAsMap();
+    MapWriter rootAsMap(Boolean caseSensitive);
```

Comment:
I think ComplexWriter itself is case sensitive or not. And this method is not needed.